Automattic/wp-calypso
# Media: Add Sort Order Dropdown

Fixes #8967. Sort-of-reboots #8959.

This PR implements the following proposal from #8967:

![screen shot 2017-01-24 at 17 38 41](https://cloud.githubusercontent.com/assets/4389/22259113/f6dabce6-e25b-11e6-8a21-dc6ef62649ee.png)

...except for the "Restore initial order" option, as discussed [here](https://github.com/Automattic/wp-calypso/issues/8967#issuecomment-307080161).

To test:
* Go to any post that includes a gallery (or create a new one). In the gallery modal's 'Edit' tab, you should see the menu as depicted above (using an '...' ellipsis symbol -- that component currently doesn't support different icons).
* Try each of its 3 options. Verify that each option does the right thing. Also verify that saving the gallery works, correctly using the current order.

Big props to @alisterscott whose code from #8959 I'm re-using here.
Also props to @folletto for helping me figure out a number of styling issues.
